yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/retrieve/WatchlistScreeningSummaryResponse.java

```diff
@@ -0,0 +1,10 @@
+package com.yoti.api.client.docs.session.retrieve;
+
+public class WatchlistScreeningSummaryResponse extends WatchlistSummaryResponse<WatchlistScreeningSearchConfigResponse> {
+
+    @Override
+    public WatchlistScreeningSearchConfigResponse getSearchConfig() {
```

Comment:
Why do we need this, if the base class already returns it?

Reply:
The base class returns a generic `Config extends WatchlistSearchConfigResponse` which is an abstract class with default package scope (meaning the _relying business_ can't use it).  We should consider making it public, or use this method